### DISCUSSION TITLE
eslint-plugin-clarity-adoption with eslint-angular

### DISCUSSION THREAD

**Discussion by jerone** [vmware-clarity/core#54] [Ideas]
I found the [eslint-plugin-clarity-adoption](https://github.com/vmware-clarity/core/tree/d1ab6b20a7d90a9ca796c096486bea29875c6cbc/packages/eslint-plugin-clarity-adoption).
However, it uses the parser from `@typescript-eslint/parser`. 
Would it be possible to support the `@angular-eslint` parser?

**Comment by bbogdanov** [comment]
Hi @jerone,

I believe it could be done but unfortunately we frozen the development on it and we don't know if we are going to get back on it any time soon.

I'll make sure to ping you here when we resume the work on it.
